drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -454,7 +474,7 @@ local function do_configure(driver, device)
     end
   end
 
-  if profile_name then
+  if not device:get_field(IS_AQARA_SWITCH_DEVICE) and profile_name then
```

Comment:
I think I'd prefer leaving the configure logic as-is, and add a more general SWITCH_DEVICE_PROFILED with an early return to the top of the function. This SWITCH_DEVICE_PROFILED field can be set for all devices using initialize_switch, not just aqara devices.
 
```suggestion
local function do_configure(driver, device)
  if device:get_field(SWITCH_DEVICE_PROFILED) then
     return
  end
 local energy_eps = embedded_cluster_utils.get_endpoints(device, clusters.ElectricalEnergyMeasurement.ID)
  ...
  if profile_name then
```